nodejs/week1/meal_sharing/src/backend/routes/cheap_meals.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const router = express();
+
+const meals = require("../data/meals.json");
+const review = require("../data/reviews.json");
+
+const cheapMeal = meals.filter(itemMeal => {
```

Comment:
`filter` returns `undefined`, if the `itemMeal.price` >= 70. It doesn't break code, but is a bit bad practice (the function is expected to return boolean). 

You could solve the issue above and reduce the size of your code to just return `itemMeal.price < 70`, as this operation already returns boolean.